test/safira_web/controllers/attendee_controller_test.exs

```diff
@@ -0,0 +1,64 @@
+defmodule SafiraWeb.AttendeeControllerTest do
+  use SafiraWeb.ConnCase
+
+  setup %{conn: conn} do
+    {:ok, conn: put_req_header(conn, "accept", "application/json")}
+  end
+
+  describe "delete attendee" do
+    test "with valid token" do
+      user = create_user_strategy(:user)
+      attendee = build(:attendee) |> Map.put(:user_id, user.id) |> insert
+      %{conn: conn, user: _} = api_authenticate(user)
+      conn = conn
+        |> delete(Routes.attendee_path(conn, :delete, attendee.id))
+        |> doc()
+      assert json_response(conn, 204) == ""
+    end
+
+    test "with valid token and with badges" do
+      user = create_user_strategy(:user)
+      attendee = build(:attendee) |> Map.put(:user_id, user.id) |> insert
+      badge = insert(:badge)
+      redeem = build(:redeem)
```

Comment:
Only inserting and not using the result. You can create a strategy for this, prefix the variable with `_` (prefered), or remove the variable